openapi2jsonschema/command.py

```diff
@@ -151,6 +151,8 @@ def default(output, schema, prefix, stand_alone, kubernetes, strict):
 
     info("Generating individual schemas")
     for title in data['definitions']:
+        group = title.split('.')[-3].lower()
+        version = title.split('.')[-2].lower()
```

Comment:
This will need to be called something else as `version` now reflects swagger/openapi version.

I just changed my fork of this to use `api_version` and it works well